src/app/commands/scoap-squad/ScoapSquad.ts

```diff
@@ -0,0 +1,122 @@
+import {
+	// ApplicationCommandPermissionType,
+	CommandContext,
+	CommandOptionType,
+	SlashCommand,
+	SlashCreator,
+} from 'slash-create';
+// import roleIDs from '../../service/constants/roleIDs';
+import ServiceUtils from '../../utils/ServiceUtils';
+import CreateNewScoapPoll from '../../service/scoap-squad/CreateNewScoapPoll';
+import ValidationError from '../../errors/ValidationError';
+
+module.exports = class ScoapSquad extends SlashCommand {
+	constructor(creator: SlashCreator) {
+		super(creator, {
+			name: 'scoap-squad',
+			description: 'Create or delete a SCOAP Squad request',
+			guildIDs: process.env.DISCORD_SERVER_ID,
+			options: [
+				{
+					name: 'assemble',
+					type: CommandOptionType.SUB_COMMAND,
+					description: 'Create a SCOAP Squad request',
+					options: [],
+					// options: [
```

Comment:
Can these comments be removed since I don't think it's in use?

Reply:
removed the options. Should this command have any particular permissions? Just kept this commented out for now until clarified.

Reply:
I think the scoap squad is best suited for project managers at bankless dao. I don't think we have a specific role for that @nonsensecodes do you think we should limit this for level 2+ contributors?

Reply:
I thought I saw a conversation around this in the DAO somewhere, or was I dreaming? I can see Level 1s using this as a means to onboard users with Guest Passes onto projects, but I don't know what the actual use case may be outside that. I think limiting it to Level 2+ contributors may make assembling a squad more focused.

Reply:
ok, I am resolving this for now until clarified. This is a quick change any time.